src/features/receive/Receive.tsx

```diff
@@ -42,6 +42,11 @@ export interface IReceiveProps {
   transactions?: ITransactionsProps[]
 }
 
+const receiveTab = {
```

Comment:
I think it's better to use enum for this one? what do you think?

Reply:
Updated